galaxy_ng/app/access_control/fields.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth.models import Permission
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from galaxy_ng.app.models import auth as auth_models
+
+
+class GroupPermissionField(serializers.Field):
```

Comment:
DRF fields are usually used to validate input data and convert it into internal representation. Mixing validation and business logic (e.g. resolving groups into database objects) may lead to tight binding between data validation and business logic layers, uncontrolled number of implicit SQL queries) and being hard to debug and re-use.